raphtory/src/algorithms/metrics/balance.rs

```diff
@@ -58,7 +58,7 @@ fn balance_per_node<'graph, G: GraphViewOps<'graph>, GH: GraphViewOps<'graph>, C
                 prop.temporal().get(name).map(|val| {
                     val.values()
                         .into_iter()
-                        .map(|valval| valval.into_f64().unwrap_or(0.0f64))
+                        .map(|valval| valval.as_f64().unwrap_or(0.0f64))
```

Comment:
Should this algorithm have a check for the type of the property instead of just returning all 0 if it happens to be a string or something else that can't be converted to float?

Reply:
Another alternative is that edge weights default to 1. @miratepuffin @Alnaimi- any thoughts?